.github/workflows/release.yaml

```diff
@@ -0,0 +1,69 @@
+# When pushing to main, we run checks and then release.
+
+name: Check and Release
+on:
+  push:
+    branches:
+    - main
+permissions:
+  contents: read
+env:
+  GENERATED_PACKAGE_DIR: dist
+  ARTIFACT_NAME: artifact-ubuntu-latest-3.10
+
+jobs:
+  build:
+    uses: ./.github/workflows/build.yaml
+    permissions:
+      contents: write
+    secrets: inherit
+
+  provenance:
+    needs: build
+    permissions:
+      actions: read
+      id-token: write
+      contents: read
+    uses: slsa-framework/slsa-github-generator/.github/workflows/generator_generic_slsa3.yml@d995948e8d53cc639c0d3ef69db31dbc243519c4
```

Comment:
Hmmm… interesting: you can reach directly into another repo’s Action `.yaml` and run it 🤔

They don’t have an actual action we could work with?

Reply:
> Hmmm… interesting: you can reach directly into another repo’s Action `.yaml` and run it thinking
> 
> They don’t have an actual action we could work with?

Because it's implemented as a reusable workflow I think.